src/main/java/minicraft/level/LevelGen.java

```diff
@@ -47,7 +51,7 @@ public LevelGen(int w, int h, int featureSize) {
 					double c = sample(x, y + stepSize); // Fetches the next value down, possibly looping back to the top of the column.
 					double d = sample(x + stepSize, y + stepSize); // Fetches the value one down, one right.
 
-					/**
+					/*
```

Comment:
This makes the comment javadoc. Don't remove it.

Reply:
The features of a JavaDoc is not used here.